ts/features/itwallet/presentation/remote/machine/actions.ts

```diff
@@ -32,6 +32,12 @@ export const createRemoteActionsImplementation = (
     });
   },
 
+  navigateToClaimsDisclosureScreen: () => {
+    navigation.navigate(ITW_REMOTE_ROUTES.MAIN, {
```

Comment:
The previous screen is temporary, we can safely replace it.
```suggestion
    navigation.replace(ITW_REMOTE_ROUTES.MAIN, {
```